mvt/provider.go

```diff
@@ -1,11 +1,20 @@
 package mvt
 
-import "github.com/terranodo/tegola"
+import (
+	"context"
 
-//Provider is the mechinism by which the system talks to different data providers.
+	"github.com/terranodo/tegola"
+)
+
+var ErrCanceled = context.Canceled
+
+//Provider is the mechanism by which the system talks to different data providers.
 type Provider interface {
 	// MVTLayer returns a layer object based
 	MVTLayer(layerName string, tile tegola.Tile, tags map[string]interface{}) (*Layer, error)
```

Comment:
I partially agree. I think we should keep the function named MVTLayer but add the context to the function signature. No need to name it MVTLayerWithContext unless we also support without context. I think this naming convention should be applied throughout.